src/Microsoft.ComponentDetection.Orchestrator/Orchestrator.cs

```diff
@@ -68,7 +68,7 @@ public async Task<ScanResult> LoadAsync(string[] args, CancellationToken cancell
         var reloadableLogger = (ReloadableLogger)Log.Logger;
         reloadableLogger.Reload(configuration =>
             configuration
-                .WriteTo.Console()
+                .WriteTo.Console(standardErrorFromLevel: LogEventLevel.Debug)
```

Comment:
This writes everything to `stderr`? I'd be more comfortable if we wrote to `stdout` when `PrintManifest` was `null` or `false`, and only wrote to `stderr` when `PrintManifest` was set to `true`. Some environments consider any output on `stderr` to be a fail condition, and unconditionally writing to `stderr` may be a breaking change.